src/pages/VaccineModulePage.js

```diff
@@ -0,0 +1,264 @@
+  );
+
+  /* Render fridge name and chevron-down icon if icon not selected fridge */
+  renderFridgeName = (fridge, isFridgeSNotelected) => {
+    const onPress = () => this.setState({ selectedFridgeCode: fridge.code });
+
+    const { fridgeInfoSectionStyle, fridgeNameTextStyle } = localStyles;
+
+    return (
+      <TouchableOpacity style={fridgeInfoSectionStyle} onPress={onPress}>
+        {!isFridgeSNotelected && this.renderIcon('chevron-down')}
+        <Text style={fridgeNameTextStyle}>{fridge.description}</Text>
+      </TouchableOpacity>
+    );
+  };
+
+  /* Temperature with celcius symbol based no passed fontSize */
+  renderTemperature = (fontSize, temperature) => {
+    const { greyTextStyleLarge } = localStyles;
+    return (
+      <View style={{ flexDirection: 'row' }}>
+        <Text style={[greyTextStyleLarge, { fontSize }]}>{temperature}</Text>
+        <Text style={[greyTextStyleLarge, { fontSize: fontSize * 0.7 }]}>℃</Text>
+      </View>
+    );
+  };
+
+  /* Render 'Breach: {num} Exposure: {fromTemp} to {toTemp} */
+  renderFridgeExtraInfo = fridge => {
+    const numberOfBreaches = fridge.getNumberOfBreaches();
+    const hasBreaches = numberOfBreaches > 0;
+    const { minTemperature, maxTemperature } = fridge.getTemperatureExposure();
+
+    const { greyTextStyleSmall, greyTextStyleLarge, fridgeInfoSectionStyle } = localStyles;
+    const extraSectionStyle = { flexDirection: 'row', justifySelf: 'flex-end', marginRight: 10 };
+
+    return (
+      <View style={[fridgeInfoSectionStyle, extraSectionStyle]}>
+        {hasBreaches && (
+          <View style={[fridgeInfoSectionStyle, { marginRight: 10 }]}>
+            <Text style={greyTextStyleSmall}>Breaches:</Text>
+            <Text style={greyTextStyleLarge}>{numberOfBreaches}</Text>
+          </View>
+        )}
+        <Text style={greyTextStyleSmall}>Exposure:</Text>
+        {this.renderTemperature(20, minTemperature)}
+        <Text style={greyTextStyleSmall}>to</Text>
+        {this.renderTemperature(20, maxTemperature)}
+      </View>
+    );
+  };
+
+  /* Render 'Breach: {num} Exposure: {fromTemp} to {toTemp} */
+  renderFridgeStock = fridge => {
+    const fridgeStock = fridge.getTotalStock();
+    const hasStock = fridgeStock > 0;
+    const { navigateTo } = this.props;
+
+    const onPress = () =>
+      navigateTo('manageVaccineItems', 'Manage Vaccine Items', {
+        initialLocation: fridge,
+      });
+
+    const { fridgeInfoSectionStyle, greyTextStyleLarge, greyTextStyleSmall } = localStyles;
+    const fontStyleLarge = { ...greyTextStyleLarge, color: SUSSOL_ORANGE };
+    const fontStyleSmall = { ...greyTextStyleSmall, color: SUSSOL_ORANGE };
+
+    if (!hasStock) return <Text style={fontStyleSmall}>Empty Fridge</Text>;
+    return (
+      <TouchableOpacity style={fridgeInfoSectionStyle} onPress={onPress}>
+        <Text style={fontStyleSmall}>Total Stock:</Text>
+        <Text style={fontStyleLarge}>{fridgeStock}</Text>
```

Comment:
need to wrap in % based views so that they're even for 100's of stock and dozens of stock for each fridge

Reply:
referenced at the bottom of PR